enhancements/update/update-status-api.md

```diff
@@ -0,0 +1,398 @@
+1. The aggregation layer `.status.{controlPlane,workerPools[]}` reports higher-level information about the update through this layer, serving as the USC's interpretation of all insights.
+1. The outermost layer, `.status.conditions`, is used to report operational matters related to the USC itself (the health of the controller and gathering the insights, not the health of the update process).
+
+We do not expect users to interact with `UpdateStatus` resources directly; the API is intended to be used mainly by tooling. A typical `UpdateStatus` instance is likely to be quite verbose.
+
+#### Update Insights
+
+Update Insights are units of update status/health information. There are two types of update insights: status insights and health insights.
+
+##### Update Status Insights
+ 
+Update Status Insights are directly tied to the update process, regardless of whether it is proceeding smoothly or not. Status Insights expose the status of a single resource that is directly involved in the update process, usually a resource that:
+
+1. Either has a notion of "being updated," such as a `Node` or `ClusterOperator`
+1. or represents a higher-level abstraction, such as a `ClusterVersion` resource (represents the control plane) or `MachineConfigPool` (represents a pool of nodes).
+
+Initially, the USC will produce status insights for `ClusterVersion`, `ClusterOperator`, `MachineConfigPool`, and `Node` resources.
+
+##### Update Health Insights
+
+Update Health Insights report a state or condition in the cluster that is abnormal or negative and either affects or is affected by the update. Ideally, none would be generated in a standard healthy update. Health insights should communicate a condition that requires immediate attention by the cluster administrator and should be actionable. Links to resources helpful in the given situation should accompany health insights. Health insights can reference multiple resources involved in the situation. For example, a health insight exposing a failure to drain a node would reference the `Node`, the `Pod` that fails to be drained, and the `PodDisruptionBudget` that protects it. Individual resources can be involved in multiple insights.
+
+#### Update Status Controller
+
+The Update Status Controller (USC) is a new component in OpenShift that will be deployed directly by the CVO and is being treated as its operand. This is uncommon in OpenShift, as the CVO typically deploys only second-level operators as its operands. In this model, the USC (providing cluster functionality) would normally be an operand, and we would need to invent a new cluster operator to manage it. Because the CVO is directly involved in updates, such an additional layer does not have value.
```

Comment:
nit: might be worth inlining or linking your your:

> Adding this layer can be considered in the future if the proposed model is problematic.

here, so folks reading through the doc in order understand that you're deferring this layer for now, and not designing anything that would make it hard to add this layer if future folks decide it would have sufficient value.

Reply:
Sounds good, addressed in https://github.com/openshift/enhancements/pull/1701/commits/353a4b5c5d9f48d7da62889dc6b055a74d54c84d